app/src/components/market/sections/market_pooling/scalar_market_pool_liquidity.tsx

```diff
@@ -571,6 +572,8 @@ export const ScalarMarketPoolLiquidity = (props: Props) => {
     }
   }
 
+  const { tokenPrice } = useTokenPrice(getToken(networkId, 'omn').address)
```

Comment:
I think token address should be lowercase here like you did with the others, could maybe update `useTokenPrice` to make addresses lowercase automatically so we don't have to worry about it in the future